docs/samples/Microsoft.ML.Samples/Static/FastTree.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+        // the alignment of the usings with the methods is intentional so they can display on the same level in the docs site. 
+        using Microsoft.ML.Runtime.Data;
+        using Microsoft.ML.Runtime.FastTree;
+        using Microsoft.ML.StaticPipe;
+        using System;
+        using System.Linq;
+
+// NOTE: WHEN ADDING TO THE FILE, ALWAYS APPEND TO THE END OF IT. 
+// If you change the existinc content, check that the files referencing it in the XML documentation are still correct, as they reference
+// line by line. 
+namespace Microsoft.ML.Samples.Static
+{
+    public partial class TrainersSamples
+    {
+        public static void FastTreeRegression()
+        {
+            // Downloading a regression dataset from github.com/dotnet/machinelearning
+            // this will create a housing.txt file in the filsystem this code will run
+            // you can open the file to see the data. 
+            string dataFile = SamplesUtils.DatasetUtils.DownloadHousingRegressionDataset();
+
+            // Creating the ML.Net IHostEnvironment object, needed for the pipeline
+            var env = new LocalEnvironment(seed: 0);
```

Comment:
> var env = new LocalEnvironment(seed: 0); [](start = 11, length = 41)

why for transforms examples we use mlContext, and here we create LocalEnvironment?

Reply:
thanks for spotting it. I updated them. 

---
In reply to: [228308134](https://github.com/dotnet/machinelearning/pull/1321#discussion_r228308134) [](ancestors = 228308134)